.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+  
+  Before submitting a pull request, please make sure the following is done:
+
+  1. Clone [the repository](https://github.com/dfe-analytical-services/explore-education-statistics) and create your branch from `dev`.
+  2. Run `npm ci && npm run bootstrap` in the repository root.
+  3. If you've fixed a bug or added code that should be tested, add tests!
+  4. Ensure the test suites passes (`npm run test`).
+-->
+
+## Summary
+
+<!--
+ Explain the **motivation** for making this change. What existing problem does the pull request solve?
+-->
+
```

Comment:
Could have other sections with heading/descriptions like:

- Related changes - Any changes that were closely related to this change but not directly part of the main work e.g. refactoring, fixes, etc.
- Screenshots - Add some screenshots if it helps illustrate the change made